app/api/v1/router.py

```diff
@@ -1,7 +1,8 @@
 from fastapi import APIRouter
-from app.api.v1.endpoints import auth, user
+from app.api.v1.endpoints import auth, user, recommended_challenge
 
 router = APIRouter()
 
 router.include_router(auth.router, prefix="/auth", tags=["auth"])
 router.include_router(user.router, prefix="/users", tags=["users"])
+router.include_router(recommended_challenge.router, prefix="/recommend-challenges", tags=["RecommendedChallenges"])
```

Comment:
Recommend Challenge 테이블이 유저와 카테고리에 종속되어 있기 때문에 endpoint에서도 그 관계가 드러나게 아래 처럼 prefix를 지정하시면 좋을 거 같습니다!
```suggestion
router.include_router(recommended_challenge.router, prefix="/users/{user_id}/categories/{category_id}/recommend-challenges", tags=["RecommendedChallenges"])
```